src/main/java/com/coveo/saml/SamlClient.java

```diff
@@ -431,11 +434,12 @@ public static SamlClient fromMetadata(
   }
 
   private void validateResponse(Response response) throws SamlException {
-    try {
-      new ResponseSchemaValidator().validate(response);
-    } catch (ValidationException ex) {
-      throw new SamlException("The response schema validation failed", ex);
-    }
+    // TODO: Not sure how to upgrade this check.
+    //    try {
+    //      new ResponseSchemaValidator().validate(response);
```

Comment:
Not sure if removing this check causes any security issue... I wasn't able to find the OpenSAML 3 equivalent for it.